setup.cfg

```diff
@@ -7,6 +7,6 @@ with-doctest=1
 doctest-extension=rst
 
 [build_sphinx]
-source-dir = docs/
-build-dir  = docs/_build
+source-dir = doc/
```

Comment:
Ok, least surprise in the doc builds. Travis will do the `cd doc && make html` to build from the Makefile path. In CircleCI `python setup.py build_sphinx` will do it from the project root but build in our default location `doc/_build/html`, additionally `sphinx-build is called for 2 different themes that output in different directories.